File: dataset_reduzido/avatar/Python/Code/atcoder_ABC110_C.py
import collections
def run ( s , t ) :
    ss = sorted ( list ( collections.Counter ( s ).values ( ) ) )
    tt = sorted ( list ( collections.Counter ( t ).values ( ) ) )
    if ss == tt :
        return 'Yes'
    else :
        return 'No'
def main ( ) :
    s = input ( )
    t = input ( )
    print ( run ( s , t ) )
if __name__ == '__main__' :
    main ( )

File: output/Llama-2-100/avatar/Python/Java/atcoder_ABC110_C.java
import java.util.Collections;
public class atcoder_ABC110_C {
    public static void main(String[] args) {
        String s = input();
        String t = input();
        List<String> ss = new ArrayList<String>(Collections.frequency(s.split(""),""));
        List<String> tt = new ArrayList<String>(Collections.frequency(t.split(""),""));
        if (ss.equals(tt)) {
            System.out.println("Yes");
        } else {
            System.out.println("No");
        }
    }
}


